src/test/java/unit/CromwellConfig.java

```diff
@@ -41,8 +41,9 @@ void cromwellGenerateConfigCustomDir() throws IOException {
     // `terra workspace set --id=$id`
     TestCommand.runCommandExpectSuccess("workspace", "set", "--id=" + getUserFacingId());
 
-    // `terra cromwell generate-config --path=build/cromwell.conf`
-    TestCommand.runCommandExpectSuccess("cromwell", "generate-config", "--dir=build");
+    // `terra cromwell generate-config --dir=build --google-bucket-name`
```

Comment:
Can you test the error cases, where someone specifies neither bucket flag and the case where they specify both?

Reply:
When neither is specified: 
```
Error: Missing required argument (specify one of these): (--workspace-bucket-name=<workspace_bucket_name> | --google-bucket-name=<google_bucket_name>)
```

When both are specified:
```
Error: --workspace-bucket-name=<workspace_bucket_name>, --google-bucket-name=<google_bucket_name> are mutually exclusive (specify only one)
```

And in both cases the usage instructions are also printed.

Reply:
Sorry, I meant that it would be great for those behaviors to include tests, like:

https://github.com/DataBiosphere/dsub/blob/main/test/integration/unit_flags.sh#L43

Reply:
Ok! Finally added the tests. The tests are currently disabled due to IAM propogation issues but I was able to run it successfully locally. Output log of test is here: https://paste.googleplex.com/5476724874674176. @mbookman This is ready for another review.